docs/openjsf/application.md

```diff
@@ -0,0 +1,193 @@
+- enable and support our efforts to expand into providing software to our community
+- Facilitate collaboration and promote adoption. We struggle to get introductions to or interface with high profile users (such as VSCode)
+
+## Statement of alignment with OpenJS Foundation charter and mission
+
+> Please refer to [the Cross Project Council's Charter](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CPC-CHARTER.md).
+> Please keep your answer to less than 250 words.
+
+"We believe that it is essential to provide a neutral home for critical projects, with common principles of technical governance and accountability." - part of the OpenJSF mission statement.
+
+We, the JSON Schema collaborators, also believe this.
+
+We feel the approach laid out in the CPC charter is reasonable.
+
+We identify with the need for collaboration between JavaScript, web solutions, and related technologies.
+
+
+## Impact and users of the project
+> What is the ecosystem impact of the project?
+
+JSON Schema has far reaching impact in the JS ecosystem. It's used by libraries such as eslint and webpack.
+VSCode (and other IDEs) use JSON Schema to validate configuration files.
+GitHub reports roughly 9k dependent packages and over 9 million dependent repositories for the most popular implementation of JSON Schema (AJV).
+
+> Who are the biggest users of the project? (_if they are not JavaScript space, then will it ever expand to JavaScript space_)
+
+Libraries: eslint and webpack.
+Specifications: OpenAPI, AsyncAPI.
+Editors: Visual Studio Code, Eclipse IDE, Jetbrains IDEs.
+Organisations: Microsoft, Google, Amazon, Mozilla, SAP, IBM, HM Government, NIH, NASA, The Deutsche Bahn.
+
+Additionally, as of OpenAPI 3.1, all these organizations: https://www.openapis.org/membership/members
+
+> What is the potential long term impact and adoption?
+
+We are at a critical juncture of JSON Schema evolution.
+The extension mechanism, JSON Schema Vocabularies, allows for the standardised extension of JSON Schema to cater to non-validation use cases.
+
+Use cases such as code generation, form generation, database generation (and validation), have evolved, but are non-standard. We're already working with existing open source projects and major organisations to facilitate standardising these use cases.
+
+- How does it differ from other projects?
+
+JSON Schema is a domain-specific declarative language for validating and annotating JSON data.
+
+It cannot be compared to OpenAPI. OpenAPI 3.1 now defers the schema definitions to the most recent release of JSON Schema[1]. Nor can it be compared to JSON-LD, which is designed for linked data. Nor can it be compared to schema.org which provides data semantics.
+
+A possible comparison is to JSON Type Definition (RFC 8927), which is designed for code generation.
+By way of comparison, JSON Schema has an extensive test suite, many more implementations and languages supported, and has vastly more use in the ecosystem.
```

Comment:
I think this overstates JTD as an alternative to JSON Schema. JTD is in the same category as Protobuf and Avro. It's a DDL, it's not for validation.